tic-tac-toe/src/components/info-bar/info-bar.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './info-bar.css';
+
+const GameInfo = ({moves, onJumpTo, outerClassName}) => {
+    const innerClassName = "info-bar";
+    const className = `${outerClassName} ${innerClassName}`
+    const movesItem = moves.map((step, move) => {
+            return (
+                <li 
+                    className={innerClassName + "__item"}
+                    key={move}
+                >
+                    <button 
+                        className={innerClassName + '__button'}
+                        onClick={ ()=>{ onJumpTo(move) } }
+                    >
+                     {move} step
+                    </button>
+                </li>
+                
+            )
+	    })
+    return (
+        <div className={className}>
+            <ul className={innerClassName + '__list'}>
```

Comment:
It's overkill here. Just add a className to your component.

Reply:
I fixed this.
```suggestion
<div className={classes.Bar}>
      <ul className={classes.List}>
        {movesItem}
      </ul>
 </div>
```